src/_cffi_src/openssl/ec.py

```diff
@@ -208,7 +213,11 @@
     int nid;
     const char *comment;
 } EC_builtin_curve;
-typedef long point_conversion_form_t;
+typedef enum {
+    POINT_CONVERSION_COMPRESSED = 0,
+    POINT_CONVERSION_UNCOMPRESSED = 1,
+    POINT_CONVERSION_HYBRID = 2,
+} point_conversion_form_t;
```

Comment:
Addendum: I guess if we don't put values the enum will be implicitly numbered by the compiler right? So maybe it would be okay to not put values in?